verbose_debugging_reports.md

```diff
@@ -13,6 +13,9 @@ registrations](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT
 #### `source-destination-limit`
 A source is rejected due to the [destination limit](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#limiting-the-number-of-unique-destinations-covered-by-unexpired-sources).
 
+#### `source-max-event-states-limit`
+A source is rejected due to [max event states](https://wicg.github.io/attribution-reporting-api/#max-event-states) exceeding the [max trigger-state cardinality](https://wicg.github.io/attribution-reporting-api/#max-trigger-state-cardinality).
```

Comment:
I'm not sure this is the correct description of this type. In particular, I don't think the max trigger-state cardinality is relevant?